manifests/loadbalancer.pp

```diff
@@ -1492,4 +1500,27 @@
       options           => ['check', 'inter 2000', 'rise 2', 'fall 5'],
     }
   }
+
+  $tacker_api_vip = hiera('tacker_api_vip', $controller_virtual_ip)
```

Comment:
I think this needs to be hte same as other openstack services.  For example with neutron:

  $neutron_api_vip = hiera('neutron_api_vip', $controller_virtual_ip)
  if $neutron_bind_certificate {
    $neutron_bind_opts = {
      "${neutron_api_vip}:${ports[neutron_api_port]}" => $haproxy_listen_bind_param,
      "${public_virtual_ip}:${ports[neutron_api_ssl_port]}" => union($haproxy_listen_bind_param, ['ssl', 'crt', $neutron_bind_certificate]),
    }
  } else {
    $neutron_bind_opts = {
      "${neutron_api_vip}:${ports[neutron_api_port]}" => $haproxy_listen_bind_param,
      "${public_virtual_ip}:${ports[neutron_api_port]}" => $haproxy_listen_bind_param,
    }
  }  

if $neutron {
    haproxy::listen { 'neutron':
      bind             => $neutron_bind_opts,
      collect_exported => false,
    }
    haproxy::balancermember { 'neutron':
      listening_service => 'neutron',
      ports             => '9696',
      ipaddresses       => hiera('neutron_api_node_ips', $controller_hosts_real),
      server_names      => $controller_hosts_names_real,
      options           => $haproxy_member_options,
    }
  }